src/node/chainstate.cpp

```diff
@@ -166,10 +166,11 @@ ChainstateLoadResult VerifyLoadedChainstate(ChainstateManager& chainman, const C
                                                          "Only rebuild the block database if you are sure that your computer's date and time are correct")};
             }
 
-            if (!CVerifyDB().VerifyDB(
-                    *chainstate, chainman.GetConsensus(), chainstate->CoinsDB(),
-                    options.check_level,
-                    options.check_blocks)) {
+            VerifyDBResult result = CVerifyDB().VerifyDB(
+                *chainstate, chainman.GetConsensus(), chainstate->CoinsDB(),
+                options.check_level,
+                options.check_blocks);
+            if (result == VerifyDBResult::ERROR_CORRUPTED_BLOCK_DB) {
```

Comment:
e7dfc743406bfa0cac94c640dd5eaeac4364c990: Should be switch-case to avoid missing an error value?

Reply:
changed to use `switch`.